submissions/akinbobolaFaith/dom/index.js

```diff
@@ -0,0 +1,65 @@
+var quantity = document.getElementById('quantity'),
+    price = document.getElementById('price'),
+    shipping = document.getElementById('shipping'),
+    cost = document.getElementById('cost'),
+    button = document.getElementById('myBtn')
+    unitPrice = 14000,
```

Comment:
You can't set unit prite here. You have to get it the same way you are getting the value of quantity @faithakinbobola